OpenEdXMobile/res/layout/fragment_learn.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:id="@+id/ll_learn_selection"
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/learn_selection_width"
+        android:layout_gravity="center"
+        android:layout_marginStart="@dimen/container_padding"
+        android:layout_marginEnd="@dimen/container_padding"
+        android:background="@color/white"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="UselessParent">
+
+        <TextView
+            android:id="@+id/tv_selected_item"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:fontFamily="?attr/fontSemiBold"
+            android:paddingEnd="@dimen/learn_drop_down_icon_margin"
+            android:text="@string/label_my_courses"
+            android:textColor="@color/primaryDarkColor"
+            android:textSize="@dimen/edx_large"
+            tools:ignore="RtlSymmetry" />
+
+        <ImageView
```

Comment:
Based on the [design](https://www.figma.com/file/Vaw3lfBXmOhTLKFnBedqE1/App-Navigation-Improvements?node-id=2961%3A15184) the icon is changing. 🤔